### ISSUE TITLE
sidebar going over other elements on the page

### ISSUE THREAD

**Issue by Hafizur046** [gerhardsletten/react-reader#138]
Hello, I recently found out about this package and while I was trying it out I noticed that if there is any other element on the page above the div containing `ReactReader` then the navigation sidebar goes over that content even in the collapsed state. I've uploaded a screenshot of the issue to [imgur](https://imgur.com/a/I19UoI2). I'm also including the code I used here:
```
    <div>
      <h1>Hello world! This is an example text</h1>
      <div style={{ height: "100vh" }}>
        <ReactReader
          location={location}
          locationChanged={locationChanged}
          url="https://react-reader.metabits.no/files/alice.epub"
        />
      </div>
    </div>
```

**Comment by Hafizur046** [comment]
Hi, @gerhardsletten , thank you for the response. I've created a minimal repository that has the least amount of packages to showcase the bug: [github.com/hafizur046/react-reader-bug-example](github.com/hafizur046/react-reader-bug-example)

**Comment by gerhardsletten** [comment]
@Hafizur046 think this is fixed in react-reader@1.0.2
